### ISSUE TITLE
Add support for `DEFAULT clause` of DDL.

### ISSUE THREAD

**Issue by nsega** [GoogleCloudPlatform/cloud-spanner-emulator#71]
It looks like the emulator doesn't yet support `DEFAULT (expression)` clause of DDL, such as CREATE TABLE or ALTER TABLE. 
Testing DDL clause in the Spanner emulator v1.4.2. The emulator returned the parse error "Error parsing Spanner DDL statement" like this:

```SQL
CREATE TABLE Sample (
  Id INT64 NOT NULL,
  Name STRING(255) NOT NULL DEFAULT ("sample"),
) PRIMARY KEY(Id);
``` 

- DDL with DEFAULT caluse
```sh
$ gcloud spanner databases ddl update test-db --instance test-instance --ddl='CREATE TABLE Sample (
  Id INT64 NOT NULL,
  Name STRING(255) NOT NULL DEFAULT ("sample"),
) PRIMARY KEY(Id);'

ERROR: (gcloud.spanner.databases.ddl.update) HTTPError 400: {"error":"Error parsing Spanner DDL statement: CREATE TABLE Sample (\n  Id INT64 NOT NULL,\n  Name STRING(255) NOT NULL DEFAULT (\"sample\"),\n) PRIMARY KEY(Id) : Syntax error on line 3, column 29: Expecting ')' but found 'DEFAULT'","code":3,"message":"Error parsing Spanner DDL statement: CREATE TABLE Sample (\n  Id INT64 NOT NULL,\n  Name STRING(255) NOT NULL DEFAULT (\"sample\"),\n) PRIMARY KEY(Id) : Syntax error on line 3, column 29: Expecting ')' but found 'DEFAULT'"}
```

- DDL without DEFAULT caluse
```sh
$ gcloud spanner databases ddl update test-db --instance test-instance --ddl='CREATE TABLE Sample (
  Id INT64 NOT NULL,
  Name STRING(255) NOT NULL,                   
) PRIMARY KEY(Id);'
Schema updating...done.   
```

ref: https://cloud.google.com/spanner/docs/reference/standard-sql/data-definition-language#parameters_3

**Comment by MatthewLymer** [comment]
I think having an accurate emulator is quite important for adoption of new features, my team recently decided against using `DEFAULT` because either our migration scripts would require two versions (one for emulator and one without), and hacks in our code to account for the difference in our local environment.

**Comment by uynap** [comment]
Hi Team, I'm having the same issue. Do you have any plan to support the `DEFAULT`?

**Comment by jward-premise** [comment]
I agree with @MatthewLymer. We really need the emulator to support the `DEFAULT` clause so that we can test our SQL migrations and create a local development environment that mirrors our higher environments as close as possible.

**Comment by Liu-Chunhui** [comment]
Hi team, i'm having the same problem. trying to execute 

```
ALTER TABLE TXXX ALTER CXXX SET DEFAULT (0);
```
and getting error `: Syntax error on line 1, column 53: Expecting 'OPTIONS' but found 'DEFAULT'`

**Comment by halmai** [comment]
I would also need this feature a lot.

**Comment by BrendM** [comment]
Also would love this

**Comment by sqlboy** [comment]
Please add this, kinda ridicuous.

**Comment by patrickmeiring** [comment]
I am on the Chrome Core Infra team (ldap: meiring@). The lack of emulator support is hindering our adoption of this useful new feature.

**Comment by kongl2** [comment]
please kindy add the support for this.

**Comment by psinghbay1** [comment]
We need it !  Can't use real Spanner for offline development.

Do you have a plan to support this in Emulator as well?

**Comment by gauravpurohit06** [comment]
Thank you everyone for your patience, the project is under development and it will be part of next [Emulator Release](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/releases).

**Comment by gauravsnj** [comment]
Default Values is now available in v1.4.8. Hence, closing this issue
